core/dbt/events/types.py

```diff
@@ -1827,6 +1827,16 @@ def message(self) -> str:
         return "No nodes selected!"
 
 
+@dataclass
+class CommandCompleted(InfoLevel, pt.CommandCompleted):
```

Comment:
This feels more `DebugLevel` than `InfoLevel` to me - do you feel strongly that it should be in the stdout logs?

Reply:
I have no strong opinion. I'm fine with it being a debug 👍

Reply:
Updated to a `Debug` level in https://github.com/dbt-labs/dbt-core/pull/7180/commits/92cec272ca70c276b892bdec412f46a73fe15ec0